tests/schemas/api_v2/fees.schema.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import Joi from 'joi';
+
+const feeEstimateSchema = {
+	feeEstimatePerByte: Joi.object({
+		low: Joi.number().min(0).required(),
+		medium: Joi.number().min(0).required(),
+		high: Joi.number().min(0).required(),
+	}).required(),
+	baseFeeById: Joi.object({
+		'5:0': Joi.string().required(),
+	}).required(),
+	baseFeeByName: Joi.object({
+		'dpos:registerDelegate': Joi.string().required(),
+	}).required(),
+	minFeePerByte: Joi.number().integer().required(),
+};
+
+const feeEstimateEventSchema = {
+	low: Joi.number().min(0).required(),
+	med: Joi.number().min(0).required(),
+	high: Joi.number().min(0).required(),
+	updated: Joi.number().integer().min(1).required(),
+	blockHeight: Joi.number().integer().min(1).required(),
+	blockId: Joi.string().required(),
+	baseFeeByModuleAssetId: Joi.object({
+		'5:0': Joi.string().required(),
+	}).required(),
+	baseFeeByModuleAssetName: Joi.object({
+		'dpos:registerDelegate': Joi.string().required(),
+	}).required(),
+	minFeePerByte: Joi.number().integer().required(),
+};
+
+const metaSchema = {
+	lastUpdate: Joi.number().integer().min(1).required(),
+	lastBlockHeight: Joi.number().integer().min(1).required(),
+	lastBlockId: Joi.string().required(),
+};
+
+const goodRequestSchema = {
+	data: Joi.object(feeEstimateSchema).required(),
+	meta: Joi.object(metaSchema).required(),
+	links: Joi.object().optional(),
+};
```

Comment:
These seem to be repetitive. Kindly consider removing them and using the definitions within the `httpGenerics.schema.js`.

Reply:
It's not repetitive. (Explained above)